manuscript/01-das-modell-zum-einsatz-bringen.md

```diff
@@ -41,7 +41,7 @@ gemeinsam genutzt werden kann.
 
 Mehrere Modelle sind unvermeidlich, aber wenn Code, der auf
 verschiedenen Modellen basiert, kombiniert wird, wird Software
-fehlerhaft, unzuverlässig und schweirig verständlich. Die Kommunikation
+fehlerhaft, unzuverlässig und schwer verständlich. Die Kommunikation
```

Comment:
Ich hatte gelernt, dass sich "schwer" auf das Gewicht und "schwierig" auf die Herausforderung bezieht. Vielleicht ist das aber auch nur in der Schweiz so ...

Reply:
Ok, dann passt das.

Reply:
👍😊